optuna/storages/_rdb/models.py

```diff
@@ -512,7 +568,14 @@ def where_study(cls, study: StudyModel, session: orm.Session) -> List["TrialValu
     @classmethod
     def where_trial(cls, trial: TrialModel, session: orm.Session) -> List["TrialValueModel"]:
 
-        trial_values = session.query(cls).filter(cls.trial_id == trial.trial_id).all()
+        trial_values = cls.where_trial_id(trial.trial_id, session)
+
+        return trial_values
```

Comment:
The current `where_trial` is a simple wrapper of `where_trial_id`. This API seems to be redundant, and do we have a plan to revise the model APIs in the future?

Reply:
Yes, I agree with the idea to revise the model APIs. We discussed this point above. Please leave comments if you have or concerns.

Reply:
Ah, you have already discussed the API design. Let me resolve the comment. Thanks.